### PR TITLE
Simple keygen utility for bakery

### PR DESCRIPTION
**Pull request by mhilton**
Simple utility to generate  bakery keys.

### CODE REVIEW

File: cmd/bakery-keygen/main.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+var format = flag.String("f", "json", "output format (json, go)")
+
+func main() {
+	flag.Parse()
+	kp, err := bakery.GenerateKey()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "cannot generate key: %s\n", err)
+		os.Exit(1)
+	}
+	switch *format {
```
Comment (rogpeppe) [inline cmd/bakery-keygen/main.go]: I'd suggest we keep things ultra simple here and just use JSON.
No particular need to make the output format configurable, I think.
Then we don't need to parse arguments at all.

**Comment by rogpeppe** [comment]
LGTM thanks, modulo one suggestion above.

**Comment by rogpeppe** [comment]
Still LGTM, thanks.

**Comment by jrwren** [comment]
:+1:

**Pull request merged**